earth_enterprise/src/common/performancelogger.h

```diff
@@ -74,6 +127,11 @@ class BlockPerformanceLogger {
     bool ended;
 };
 
+// Should be called at the beginning of gecombineterrain to create files
+#define OPEN_PERF_LOG_FILES() openFiles()
```

Comment:
I think it would be safer, and simpler, if you created files in the logger constructor, and let them be closed in the destructor.

Reply:
Will do.